scripts/package.json

```diff
@@ -46,8 +46,8 @@
     "jju": "^1.4.0",
     "jscodeshift": "^0.5.0",
     "json-loader": "^0.5.7",
-    "just-scripts": "~0.14.4",
-    "just-task": "~0.9.5",
+    "just-scripts": "0.14.4",
```

Comment:
intentional?

Reply:
Yes, updating the shrinkwrap brought in a new version of Just that the repo wasn't ready for. I had to pin in for now until the changes Ken made get merged in. I'll be sure to revert before merge.